svc-managers/main.go

```diff
@@ -44,8 +43,8 @@ func main() {
 	}
 
 	err := addManagertoDB()
-	if err != nil && !(strings.Contains(err.Error(), "already exists")) {
-		log.Fatalf("error while trying to add  manager details into DB: %v", err)
+	if err != nil {
+		log.Println(err)
```

Comment:
What if the system is unable to add the manager the first time? We need to panic in this case.

Reply:
added

Reply:
what will the end errors look like.

Reply:
if the manager id present, the following log we will get 
`skipped saving of duplicate data with key /redfish/v1/Managers/34ce0e72-cf7a-47bc-8277-71241dbe43a1`

if it is some other error, then it would be like,
`error while trying to add manager details into DB: <correspond error>`

**correspond error** can be connection error or marshalling error

Reply:
I want the actual log. Not your understanding

Reply:
and what is correspond error ?? I asked for full log mesage

Reply:
if the manager id present, the following log we will get
`I | skipped saving of duplicate data with key /redfish/v1/Managers/34ce0e72-cf7a-47bc-8277-71241dbe43a1`

if it is some other error, then we will panic it and it would be like,
`I | error while trying to check DB connection health: error while trying to ping InMemory DB: error while pinging DB with read connection`